docs/editor/workspaces.md

```diff
@@ -9,39 +9,91 @@ MetaDescription: Learn about Visual Studio Code workspaces
+
+*A single folder workspace opened inside VS Code*
+
+The same is true for multi-root workspaces that you can open easily by selecting the `<name>.code-workspace` file for opening.
+
+![Multi-folder workspace](images/workspaces/multi-folder-workspace.png)
 
+*A multi-root workspace opened in VS Code*
+
+**Note:** The visual difference of having a folder opened vs. opening a `.code-workspace` file can be very subtle. To give you a hint that a `.code-workspace` file has been opened, some pieces of the user interface (e.g. the root of the explorer) show an extra "(Workspace)" suffix next to the name.
+
+## Workspace state
+**bpasero>** I would leave this paragraph out as we already gave some examples of workspace state before
 List some of the metadata and state persisted per folder TBD
 
 ## Workspace settings
-
+**bpasero>** see my note in [1]
 Workspace files are stored at the project root in a `.vscode` folder. TBD
 
 ### settings
-
+**bpasero>** see my note in [1]
 Describe workspace settings, Settings UI tab, why you would use workspace settings versus user settings
 
 ### debugging
-
+**bpasero>** see my note in [1]
 Briefly describe `launch.json` and link to other docs TBD
 
 ### tasks
-
+**bpasero>** see my note in [1]
 Briefly describe `tasks.json` and link to other docs TBD
 
+[1] **bpasero>**
+I am trying to distill what we need to explain for settings, debugging and tasks in the context of explaining what a "workspace" is. My initial reaction is that all of this should have already been explained in the respective doc pages so I would not make this very detailed here to not repeat everything. However, I think one important concept is worthwhile to mention here: if you open a `.code-workspace` file (or actually when you are inside an "untitled" workspace), you get a third dimension of settings that you don't have otherwise:
+- `.code-workspace` / untitled workspace: user, workspace, folder
+- folder: user, workspace & folder (is the same)
+
 ## Multi-root workspaces
 
-You can also have more than one root folder in a VS Code workspace through a feature called [Multi-root workspaces](/docs/editor/multi-root-workspaces.md). TBD Why you would use multi-root workspaces, briefly show UI and `.code-workspace` file. Leave details to existing doc.
+[Multi-root workspaces](/docs/editor/multi-root-workspaces.md) are an advanced capability of VS Code that allow you to configure multiple distinct folders to be part of the workspace. Instead of opening a folder as workspace, you will open a `<name>.code-workspace` JSON file that lists the folders of the workspace. For example:
+
+```json
+{
+	"folders": [
+		{
+			"path": "my-folder-a"
+		},
+		{
+			"path": "my-folder-b"
+		}
+	],
+    "settings": {
+        ...
+    }
+}
+```
+
+The presence of this `.code-workspace` file makes it even possible to include workspace specific settings (or launch / task configurations) that should apply to all folders of the workspace.
+
```

Comment:
We may also want to mention that in a multi-root workspace there can be multiple .vscode folders contributing configurations which are still scoped to the specific folders.